### ISSUE TITLE
Support for Atomic Soft Tabs?

### ISSUE THREAD

**Issue by Snepsts** [nirix/atom-editor-settings#34]
I have started using the nim programming language for a school project. One quirk of the language is it requires spaces and does not allow tabs. I prefer hard tabs so I can change the length of all of my tabs whenever I feel like it. Therefore, I like to keep track of my spaces to not confuse them with tabs, so I disable Atomic Soft Tabs (it runs over 4 spaces like they're a tab if your tab length is 4). But with something like nim I'd like to run over my tab length like they're a tab.

I think this would be a really useful feature for other people that have to switch between using hard tabs and soft tabs as well.

tl;dr is it possible to add atomic soft tabs to the settings this package supports?

**Comment by nirix** [comment]
Ideally this package would support all settings that affect a language/editor view, but not all are easily set in a performant way.

That being said, I will try to put some time aside after work to add in as many missing settings as possible.

**Comment by Snepsts** [comment]
Appreciated!
